src/utils/imageOutputCuller.ts

```diff
@@ -0,0 +1,59 @@
+const LineSplitter = /\r\n?|\n/;
+const LineParser = /^[-\w]+-sha256:\w+:\s*(\w+)\s*\|.*?\|/;
```

Comment:
Example?

Reply:
When pulling `redis:latest`, it has multiple layers:
```
$ kim pull redis:latest
layer-sha256:dd0ea236b03be5127b0b7fb44f1d39f757c5381a88d650fae0a0d80b0c493020: downloading    |+++++++++++++++-----------------------|  4.0 MiB/9.6 MiB
layer-sha256:f7ec5a41d630a33a2d1db59b95d89d93de7ae5a619a3a8571b78457e48266eba: downloading    |++++----------------------------------|  3.0 MiB/25.9 MiB
…
```
(The capture isn't quite right, because it keeps clobbering the output when I try.)

Reply:
There should be 6 layers, but because of the multi-layer-ness, only 2 of the layers move from the "waiting" to the "downloading" state. The problem is that I had a fancy regex that was putting everything in capture groups, but I only needed the SHA. Unfortunately I captured the status field instead. Fixed on commit 8077ad0